python/hidet/graph/ops/definitions/conv2d/conv2d.py

```diff
@@ -10,17 +10,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from typing import List, Union, Sequence
+from math import floor
 from hidet.graph.ops.definitions.utils import Task, Operator, Tensor, TensorNode
 from hidet.graph.ops.definitions.utils import compute, input_like, normalize_stride, reduce
 
 
 class Conv2dTask(Task):
-    def __init__(self, data: TensorNode, weight: TensorNode, stride: List[int], groups: int):
+    def __init__(self, data: TensorNode, weight: TensorNode, stride: List[int], dilations: List[int], groups: int):
         # pylint: disable=too-many-locals
         n, c, h, w = data.const_shape()
         oc, wc, kx, ky = weight.const_shape()
         sx, sy = stride
-        p, q = (h - kx) // sx + 1, (w - ky) // sy + 1
+        dilx, dily = dilations
+        p, q = floor((h - dilx * (kx - 1) - 1) / sx + 1), floor((w - dily * (ky - 1) - 1) / sy + 1)
```

Comment:
```suggestion
        (h - dilx * (kx - 1) - 1) // sx + 1, (w - dily * (ky - 1) - 1) // sy + 1
```
I suggest to use the integer arthmatics instead of floor + float arthematics (/). The later method has the risk of float error like floor(1.9999999) = 1.